### PR TITLE
Add DeSci Labs to DeSci page

### PR DESCRIPTION
**Pull request by erik-descilabs**
Added DeSci Nodes and DeSci Labs to the content on the DeSci Page

Changed the reproducibility section to provide tangible examples of reproducible research directly into the section. Additionally added a link to the DeSci Labs website into the list of projects below.

### CODE REVIEW

**Comment by erik-descilabs** [comment]
Hi team,

Would someone be able to review this PR? Just wanted to check in and see if there's anything I can do to help push this forward.

Thanks,
Erik

**Comment by vincentweisser** [comment]
Looks good to me, DeSci Labs is def a key project to mention! So +1

File: src/content/desci/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ Reproducibility and replicability are the foundations of quality scientific disc
...
 
-New Web3-native tools can ensure that reproducibility and replicability are the basis of discovery. We can weave quality science into the technological fabric of academia. Web3 offers the ability to create attestations for each analysis component: the raw data, the computational engine, and the application result. The beauty of consensus systems is that when a trusted network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.
+New Web3-native tools such as [DeSci Nodes](https://beta.dpid.org/46) can ensure that reproducibility and replicability are the basis of discovery by allowing code, data, and pdfs to be linked together through reproducibility annotations. Web3 offers the ability to create attestations for each analysis component: the raw data, the computational engine, and the application result. The beauty of consensus systems is that when a trusted network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.
```
Comment (minimalsm) [inline src/content/desci/index.md:63]: We shouldn't be listing products in the body of content.

```suggestion
New Web3-native tools can ensure that reproducibility and replicability are the basis of discovery. We can weave quality science into the technological fabric of academia. Web3 offers the ability to create attestations for each analysis component: the raw data, the computational engine, and the application result. The beauty of consensus systems is that when a trusted network is created for maintaining these components, each network participant can be responsible for reproducing the calculation and validating each result.
```

**Review by minimalsm — APPROVED** [review]
Thanks for the PR @erik-descilabs 🙏

**Pull request merged**